config/webpack.config.lib.js

```diff
@@ -46,9 +46,6 @@ module.exports = {
   module: {
     rules: [
       { test: /\.tsx?$/, use: "ts-loader" },
-      // All output '.js' files will have any sourcemaps re-processed by 'source-map-loader'.
```

Comment:
Could you explain why do you need to remove `source-map-loader` here?

Reply:
Sorry didn't add this while adding back source maps. my bad.